app/src/main/java/app/MainActivity.kt

```diff
@@ -1,12 +1,114 @@
 package app
 
 import android.os.Bundle
+import android.os.StrictMode
+import android.widget.Button
 import androidx.appcompat.app.AppCompatActivity
-import app.AnimalVilla.R
+import app.AnimalVilla.R.*
+import app.DAO.IPromptDAO
+import app.DTO.Prompt
+import org.json.JSONArray
+import org.json.JSONObject
+import java.io.BufferedReader
+import java.io.InputStreamReader
+import java.net.URL
+import java.util.concurrent.LinkedBlockingQueue
+import kotlin.concurrent.thread
+
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        setContentView(layout.activity_main)
+        findViewById<Button>(id.StartButton).setOnClickListener{
```

Comment:
This looks like an incomplete feature.
Are you using this click listener anywhere?  I don't see them used.
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.